src/components/SQForm/SQFormMultiValue.tsx

```diff
@@ -1,28 +1,59 @@
 import React from 'react';
-import PropTypes from 'prop-types';
 import {Grid, Chip, TextField, makeStyles} from '@material-ui/core';
-import Autocomplete from '@material-ui/lab/Autocomplete';
-import {VariableSizeList} from 'react-window';
+import Autocomplete, {
+  AutocompleteChangeReason,
+} from '@material-ui/lab/Autocomplete';
+import {ListChildComponentProps, VariableSizeList} from 'react-window';
 import {useField, useFormikContext} from 'formik';
-import {useForm} from './useForm';
 import {usePrevious} from '@selectquotelabs/sqhooks';
+import {useForm} from './useForm';
+import {BaseFieldProps, Option, optionValue} from 'types';
+import {
+  OuterElementContextInterface,
+  OuterElementTypeProps,
+} from './SQFormAutocomplete';
+
+interface SQFormMultiValueProps extends BaseFieldProps {
+  /** options to select from */
+  children: Option[];
```

Comment:
Based on the previous propTypes, everything outside of `BaseFieldProps` should be optional

Reply:
Yep, everything except Children. It should actually be required even if the array is empty.